src/test/java/org/rulez/demokracia/pdengine/VoteModificationInvariantsObservedTest.java

```diff
@@ -43,11 +44,12 @@ public void checkInvariables(Vote vote) {
 		assertEquals(savedCountedAssurances,vote.countedAssurances);
 		assertEquals(savedIsPrivate, vote.isPrivate);
 		assertEquals(savedCreationTime, vote.creationTime);
+		assertEquals(savedCanUpdate, vote.canUpdate);
 	}
 
 	@tested_feature("Manage votes")
 	@tested_operation("modify vote")
-	@tested_behaviour("vote invariants")
```

Comment:
You are revoking a test here. don't do that.
Just add the new behaviour (I hope the analysis script can cope with that. If not, then it will be fixed.)

Reply:
I can not leave two tested_behaviour here :(
So should I create a separate test for it?